src/main/java/me/dinowernli/grpc/prometheus/Configuration.java

```diff
@@ -77,7 +89,25 @@ public Configuration withLabelHeaders(List<String> headers) {
     List<String> newHeaders = new ArrayList<>(labelHeaders);
     newHeaders.addAll(headers);
     return new Configuration(
-        isIncludeLatencyHistograms, collectorRegistry, latencyBuckets, newHeaders);
+        isIncludeLatencyHistograms,
+        collectorRegistry,
+        latencyBuckets,
+        newHeaders,
+        addCodeLabelToHistograms);
+  }
+
+  /**
+   * Returns a copy {@link Configuration} with the difference that status code label will be added
+   * to latency histogram. If latency histogram itself is disabled, this takes no effect.
+   */
+  public Configuration withCodeLabelInLatencyHistogram() {
+    return new Configuration(
+        isIncludeLatencyHistograms,
+        collectorRegistry,
+        latencyBuckets,
+        labelHeaders,
+        // addCodeLabelToHistograms: set true only if isIncludeLatencyHistograms is also true
+        isIncludeLatencyHistograms);
```

Comment:
Any reason why we can't just set this to "true"? If it's true and latency histograms are not included, then that would still (correctly) have no effect, right?

Reply:
Yes, this would be simpler, done!)